app/com/arpnetworking/rollups/ConsistencyChecker.java

```diff
@@ -691,7 +690,7 @@ public Builder setRollupSampleCount(final long value) {
 
         // This "should" be a MetricsQueryResponse, but that class isn't Serializable.
         // ("Why isn't it?" Several of the data-models contain Optional values, and Optional isn't Serializable.)
-        private final JsonNode _response;
```

Comment:
why change this to String?

Reply:
`JsonNode` is `JsonSerializable` but not `Serializable`. Frankly, the reasoning around using `JsonNode` instead of `MetricsQueryResponse` seems suspect. Perhaps, this is no longer a requirement. However, 2.10 does not make this hierarchy serializable; when I compiled it against 2.9 I was received a warning.